staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -187,6 +185,13 @@ type Config struct {
 	PublicAddress net.IP
 }
 
+type RecommendedConfig struct {
+	Config
+
+	// SharedInformerFactory provides shared informers for resources
+	SharedInformerFactory informers.SharedInformerFactory
```

Comment:
Expand the godoc explaining that is normally set via RecommendedOptions.ApplyTo.

Reply:
done